flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -310,6 +310,12 @@ class TaskManager(
 
     case FatalError(message, cause) =>
       killTaskManagerFatal(message, cause)
+
+    case RequestTaskManagerLog(requestType : LogTypeRequest) =>
+      handleRequestTaskManagerLog(sender(), requestType, currentJobManager.get)
+
+    case IsBlobServiceDefined =>
+      sender() ! (blobService.isDefined && currentJobManager.get.path.address.host.isDefined)
```

Comment:
What is the value of `path` in the local case? Maybe we should then extend the `BlobService` to simply give back a `BlobClient` which has the right connection address.